src/modules/peek/Peek.FilePreviewer/Previewers/IPreviewer.cs

```diff
@@ -15,9 +16,9 @@ public interface IPreviewer : INotifyPropertyChanged
 
         public static bool IsFileTypeSupported(string fileExt) => throw new NotImplementedException();
 
-        public Task<Size> GetPreviewSizeAsync();
+        public Task<Size> GetPreviewSizeAsync(CancellationToken cancellationToken);
```

Comment:
FYI: Usually, we make those optional and make them default to CancellationToken.None. Don't need to change it for your PR! Just thought I'd share how I've previously used them.